softvis3d.properties

```diff
@@ -1,14 +1,20 @@
 category=Visualization/Reporting
 description=Provides 3D models
 homepageUrl=http://softvis3d.com
-archivedVersions=0.5.2,0.6.0,0.7.0,0.8.0,0.8.1,0.9.0,1.0,1.0.1
-publicVersions=1.1.0
+archivedVersions=0.5.2,0.6.0,0.7.0,0.8.0,0.8.1,0.9.0,1.0,1.0.1,1.1.0
+publicVersions=1.2.0
 
 defaults.mavenGroupId=de.rinderle.softvis3d
 defaults.mavenArtifactId=softvis3d
 
+1.2.0.description=New color and filter features.
+1.2.0.sqVersions=[7.6,8.7.*]
```

Comment:
This range should end with `LATEST`... Unless you really want to make it incompatible with the current version of SonarQube?